src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -450,3 +282,129 @@ def save_model_training(
                 info_lines.append(f"{loss}: {val_res.result_mean(loss).item()}")
 
         info_file.write("\n".join(info_lines))
+
+
+# private methods
+def _get_collate_fn(arch_key: Optional[str] = None, task: Optional[Tasks] = None):
```

Comment:
noting we can get rid of this function as we no longer plan on supporting object detection flows (for ssd and yolo) in this script as it is just for image classification now

Reply:
Will get rid of this in the upcoming helpers refactor